package.json

```diff
@@ -88,19 +136,14 @@
   "peerDependencies": {
     "react-hook-form": ">=6.6.0"
   },
-  "release": {
```

Comment:
By removing this, this allows us to use semantic-release's default config, so master, beta branches will be automatically published.

In our case, beta will create 2.0.0-beta.X version

https://semantic-release.gitbook.io/semantic-release/usage/workflow-configuration#pre-release-branches